app/pages/api/organizations/index.js

```diff
@@ -8,18 +8,19 @@ import Org from '../../../models/Org';
  */
 export default async function handler(req, res) {
   try {
-    const { method } = req;
+    const { method, query } = req;
     await dbConnect();
     switch (method) {
       case 'GET':
         try {
-          const orgs = await Org.find({ isHidden: { $ne: true } });
+          const filter =
```

Comment:
Is there a reason we can't do this query on MongoDB?

Reply:
This is in MongoDB. Specifically, https://mongoosejs.com/docs/api/query.html#query_Query-sort.